README.md

```diff
@@ -104,13 +104,13 @@ curl -v -X GET "https://share.actnowcoalition.org/shareLinksByUrl?url=https://ww
 
 Generates a screenshot of the given target URL and returns the image.
 
-The target URL must contain `<div>`s with `screenshot` and `screenshot-ready` CSS classes to indicate where to capture and when the screenshot is ready to be taken, e.g.:
+The target page must contain an element with `screenshot` as a CSS class name to indicate where the
+screenshot will be captured. Because components on the target page can take time to load, the endpoint also uses specific class names to tell when the page is ready to be captured. The screenshot is captured when:
 
-```html
-<div class="screenshot">
-  <div class="screenshot-ready">{content to screenshot}</div>
-</div>
-```
+- an instance of the `act-now-component-loaded` CSS class is detected on the page
+- no instances of the `act-now-component-loading` CSS classes are detected
```

Comment:
I'm feeling a bit conflicted about using class names that are act-now-specific since (originally) the link service was built to be independent of act-now-packages components.  Anybody could use it, not necessarily just act now components.

But using act-now-component-loaded/loading makes sense from the act-now-packages perspective.  So I think it's probably fine.  We could always change it later.

Reply:
Yeah, I agree, I think this could be solved by letting them be configurable, as you mentioned below. I've opened a follow-up ticket for this https://github.com/orgs/act-now-coalition/projects/6/views/17?pane=issue&itemId=20026894